ansible/roles/iroha-docker/templates/docker-compose.yml.j2

```
@@ -35,6 +35,11 @@ services:
     volumes:
 {% if iroha_blockstore_path  %}
       - {{ iroha_volume_dir }}iroha_block_store-{{ node.human_hostname }}:{{ iroha_blockstore_path }}
+{% endif %}
+{% if iroha_use_rdb %}
+{% for node in iroha_nodes %}
+      - {{ iroha_volume_dir }}iroha_rdb_data-{{ node.human_hostname }}:{{ iroha_rdb_data_path }}
+{% endfor %}
```

```suggestion
      - {{ iroha_volume_dir }}iroha_rdb_data-{{ node.human_hostname }}:{{ iroha_rdb_data_path }}
```
you should declare volume (`iroha_rdb_data-{{ node.human_hostname }}`) in  line [L116](https://github.com/hyperledger/iroha-deploy/pull/29/files#diff-edfc266ca2396ddbb84813dc917fed98799ce983937d169740ba2e8f145ad3c1L111)

---

Ah, now I get it, thanks. Fixed